index.html

```diff
@@ -1,19 +1,32 @@
 <!doctype html>
 <html lang="en">
-<head>
+<head title="jobmatix-pixel">
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 </head>
 <body>
 <script type="text/javascript" async>
+  //Variant 1
```

Comment:
Why do we need to keep this commented ?

Reply:
there is no reason, I just keep them as example of installation code and also for testing